### PR TITLE
Added SSTU Apollo and AJ10-137

### PR DESCRIPTION
**Pull request by FFCJoseEduardo**
Needs to be tested

### CODE REVIEW

**Comment by stratochief66** [comment]
RCS on the new Apollo capsule doesn't appear to work, or at least it didn't appear to consume fuel or generate thrust.

The custom SSTU chute module didn't appear to generate drag (with FAR) @FFCJoseEduardo did it work with stock atmo? If so, my guess is that he might need to do something to cause it to also reflect increased drag with FAR. I don't think we can override his chutes since they are built into the capsule.

**Comment by stratochief66** [comment]
What are the advantages/difference between the SSTUHeatShield module and the ModuleAblator we use? If there isn't any advantage to SSTUHeatShield I would suggest a straight removal

!MODULE[SSTUHeatShield] {}

**Comment by stratochief66** [comment]
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R111

Please rephrase this as an @ . Also, @ inside the module since it already exists.

**Comment by stratochief66** [comment]
The SM engine bell moves when I try to aim the thrust, but the thrust itself doesn't seem to gimbal. interesting

**Comment by stratochief66** [comment]
Tested, worked!

https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R209

Remove this line, and the CM RCS works.

**Comment by stratochief66** [comment]
Looks pretty good, I will have to do a quick test again before I can merge.

Please skim for indentation errors, like these:
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R96
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R96
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R302
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R302
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R302

**Comment by stratochief66** [comment]
Any if anybody (@NathanKell ? ) has any idea why this part would visually gimbal, but the FX and actual through wouldn't gimbal, please let me know.

https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-58a231f424e726507f69400cd3afd542R563



@FFCJoseEduardo I need to check the assemblies of these engines, it could be that this one and the clusters don't share their thrustTransform name, but the clustered ones appear like this:
https://github.com/KSP-RO/RealismOverhaul/pull/779/files#diff-534c625c6ec30d529e704a72cff4b5a3R1341

**Comment by NathanKell** [comment]
Presumably because the thrust and FX transform(s) aren't parented to the gimbal transform?

**Comment by stratochief66** [comment]
@NathanKell 

Is that done at the mu/model level?

**Comment by NathanKell** [comment]
Yes. I would bring it up with the author; we should mirror their original configs regarding what transforms to use for what, so if it's not working for us it's not working for them either and they should fix it.
